src/components/WebexSettings/Tabs.jsx

```diff
@@ -43,6 +74,7 @@ export default function Tabs({
               <button
                 onClick={() => onSelect(tab.key)}
                 type="button"
+                tabIndex={-1}
```

Comment:
don't tabs need to also be navigated with the TAB key?

Reply:
From the documentation we have on Figma it looks like these can be navigated only with arrow keys